Frameworks/UIKit/XamlControls.h

```diff
@@ -121,6 +121,12 @@ WXCGrid* CreateLabel();
 WXCTextBlock* GetLabelTextBlock(WXCGrid* labelGrid);
 
 ////////////////////////////////////////////////////////////////////////////////////
+// ScrollViewer
+////////////////////////////////////////////////////////////////////////////////////
+WXCGrid* CreateScrollViewer();
+WXCScrollViewer* GetScrollViewer(WXCGrid* labelGrid);
```

Comment:
Well, technically the Grid *is* the ScrollView; we just can't project our ScrollView type to ObjC.  Either sounds fine to me though.